docs: update wording in diffing.md (#18027)
argoproj/argo-cd: docs/user-guide/diffing.md

The above configuration will ignore differences from all fields owned by `kube-controller-manager` for all resources belonging to this application.

> If you have a slash `/` in your pointer path, you must use the `~1` character. For example:

Reviewer: How about

```suggestion
If you have a slash `/` in your pointer path, you need to replace it with the `~1` character. For example:
```
